pandas/io/packers.py

```diff
@@ -214,6 +217,14 @@ def convert(values):
     if dtype == np.object_:
         return v.tolist()
 
+    if is_categorical_dtype(dtype):
+        return {
```

Comment:
actually leave this here (as its a dtype operation), but ALSO add ``Categorical`` directly to encode/decode. Need to add tests for that as well (e.g. just encoding/decoding a ``Categorical`` w/o a Series)